tcf_website/urls.py

```diff
@@ -29,6 +29,15 @@
     path('reviews/<int:review_id>/downvote', views.downvote),
     path('reviews/check_duplicate', views.review.check_duplicate),
     path('profile', views.profile, name='profile'),
+    path('unsave_course/<int:course_id>/<int:instructor_id>',
+         views.unsave_course, name='unsave_course'),
+    path('save_course/<int:course_id>/<int:instructor_id>',
+         views.save_course, name='save_course'),
+    path('saved/reorder', views.SavedCourseReorderingView.as_view(),
+         name='reorder_saved_courses'),
+    path('save_course/<int:course_id>/<int:instructor_id>/edit',
+         views.edit_course, name='edit_course'),
+    path('courses', views.saved_courses, name='courses'),
```

Comment:
Would courses be a little ambiguous? (since we already have `/course` which goes before the course URLs)
Maybe `/profile/courses`?

Reply:
I initially chose `courses` since the url to view user reviews is just `reviews`. It would make sense to change both though.